web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -649,7 +694,12 @@ export default class RunComparison extends ReactJS.Component {
                   (Object.keys(timeseriesDropdown).length > 0) ? (
                     <div style={{ marginBottom: 32 }}>
                       {Object.keys(tableData).map(table => (
-                        <div>
+                        <div id={'alltable' + table}>
+                          <input
+                            type="checkbox"
```

Comment:
So will this be displayed as an actual checkbox or will it be displayed as a simple button that can be pressed?

Reply:
@portante  To select the custom divisions, I am planning to have checkboxe. We can hide them initially and show them once the share button is pressed.